cmd/notify-mailer/main.go

```diff
@@ -459,7 +460,31 @@ func main() {
 	dbURL, err := cfg.NotifyMailer.DB.URL()
 	cmd.FailOnError(err, "Couldn't load DB URL")
 
-	dbMap, err := sa.NewDbMap(dbURL, sa.DbSettings{MaxOpenConns: 10})
+	conf, err := mysql.ParseDSN(dbURL)
+	cmd.FailOnError(err, "Couldn't parse DB URL as DSN")
+
+	// Transaction isolation level READ UNCOMMITTED trades consistency for
+	// performance.
+	const (
+		readUncommitted = "'READ-UNCOMMITTED'"
+		txIsolation     = "tx_isolation"
+	)
+	if len(conf.Params) == 0 {
+		conf.Params = map[string]string{
+			txIsolation: readUncommitted,
+		}
+	} else {
+		conf.Params[txIsolation] = readUncommitted
+	}
```

Comment:
You could do away with the duplication (and associated `const` block) by doing:
```suggestion
	if len(conf.Params) == 0 {
		conf.Params = make(map[string]string)
	}
	conf.Params["tx_isolation"] = "'READ-UNCOMMITTED'"
```